docs/dev/test-configuration-options.md

```diff
@@ -275,9 +275,9 @@ Enables Performance Capture feature. Sauce Performance Testing can be enabled by
 
 Specifies the screen resolution to be used during your test session. Default screen resolution for Sauce tests is `1024x768`.
 
-:::note
-You cannot set screen resolution on Windows 7 with IE 9.
-
+:::note notes
+- We recommend that you specify the [`platformName`](#platformname) when you set the screen resolution. 
+- Additionally, if you want to set the screen resolution on Windows, we recommend that you use Windows 8 or newer (e.g., Windows 10).
 :::
```

Comment:
I would suggest that these two lines just be merged into one:

"We recommend that you specify the [`platformName`](#platformname) to Windows 8 or newer (e.g., Windows 10) if you want to set the screen resolution on Windows"

Reply:
Done - thank you!